### DISCUSSION TITLE
Programming large NOR flash modules

### DISCUSSION THREAD

**Discussion by jaskamakkara** [sanni/cartreader#60] [Need help]
Hi Sanni,

I was wondering if it's possible to use this device to also programme some larger NOR flash devices. Something like the S29GL01GT (which is 1Gbit in size). I see that this device is capable of programming the smaller S29GL032 so I feel like it should be possible.

Failing that, do you know of a relatively cheap method for programming these sorts of devices? It's too big for my budget programmer and making an Arduino sketch for something like this is a little out of my league (I have trouble even with EPROMs).

Thanks

**Comment by sanni** [comment]
You would need to create an adapter PCB to connect the flash to the Arduino and then modify the existing flash functions in the code to support the S29GL01GT. So it's possible but it's also alot of work.

There is a budget friendly programmer that gets recommended quite a lot and does support S29GL01GT: https://www.embeddedcomputers.net/products/FlashcatUSB_XPORT/

**Reply by jaskamakkara** [reply]
OK, well perhaps I will look into the product you linked there - thanks very much for that I wasn't aware of it before.
